src/index.js

```diff
@@ -1,14 +1,47 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
+// add to do list
+const toDoList = [];
+
+const taskOne = {
+  description: 'make the bed',
+  completed: false,
+  index: 0,
+};
+
+toDoList.push(taskOne);
+
+const taskTwo = {
+  description: 'walk the dog',
+  completed: false,
+  index: 1,
+};
+toDoList.push(taskTwo);
```

Comment:
**[OPTIONAL]** 
- [ ] As the previous reviewer suggested, you can make your array as shown below instead of using push method for an array because that below approach is professional 👇
 
```
const todoList = [
  {
    description: 'make the bed',
    completed: false,
    index: 0,
  },
  {
     description: 'walk the dog',
    completed: false,
    index: 1,
  },
  {
    etc......
  }
];
```